stdlib/src/builtin/simd.mojo

```diff
@@ -105,6 +105,29 @@ fn _unchecked_zero[type: DType, size: Int]() -> SIMD[type, size]:
     }
 
 
+@always_inline
+fn _is_index_32bit() -> Bool:
+    @parameter
+    if sizeof[DType.index]() == sizeof[DType.int32]():
+        return True
+    else:
+        return False
```

Comment:
I don't think the param if is necessary, as the function itself is only called in a param if so it's guaranteed to be compile time.

Reply:
it should probably go somewhere else aswell, like `dtype.mojo` or `info.mojo`